src/cache.ts

```diff
@@ -64,7 +64,7 @@ export class CachedFetcher<T> {
     ]);
   }
 
-  private get currentValue(): T | undefined {
+  get currentValue(): T | undefined {
```

Comment:
I'm not sure how to spy/mock this without making it public. If anyone knows, please let me know

Reply:
Turns out one just needs to cast the prototype to `any`, https://github.com/yearn/yearn-sdk/pull/243/commits/5ae583054c08dbbe08f74f5f697bd1b8b760a039